templates/portal/apps.html

```diff
@@ -1,14 +1,14 @@
 {% extends 'base.html' %}
 
-{% block title %}Apps{% endblock %}
+{% block title %}Apps and models{% endblock %}
```

Comment:
```suggestion
{% block title %}Apps and Models{% endblock %}
```

Reply:
We don't capitalize other main menu items so we will keep it like this for consistency.